### ISSUE TITLE
about three additional labels

### ISSUE THREAD

**Issue by wwk-png** [graspnet/anygrasp_sdk#28]
Thank you for your contributions. As I was reading through the paper, I noticed that three additional labels have been added to the training data. I was wondering if these three new annotations haven't been made available to the public yet?

**Comment by chenxi-wang** [comment]
If you mean the labels of graspness and stable score, they have not been released, but can be easily generated by yourself according to the paper. By the way, you can also refer to the [unofficial implementation](https://github.com/graspnet/graspness_unofficial) for graspness generation.
